src/main/groovy/com/avast/gradle/dockercompose/ComposeExtension.groovy

```diff
@@ -39,6 +39,7 @@ class ComposeExtension {
     String dockerExecutable = 'docker'
 
     String dockerComposeWorkingDirectory = null;
+    int dockerComposeStopTimeout = 10

```

Comment:
I would prefer if we had `Duration` instead of `int` ([Primitive Obsession](http://blog.ploeh.dk/2011/05/25/DesignSmellPrimitiveObsession/)). Could you please change it? Thanks!